server.js

```diff
@@ -14,10 +15,55 @@ const welcomeMessage = {
 //This array is our "data store".
 //We will start with one message in the array.
 //Note: messages will be lost when Glitch restarts our server.
-const messages = [welcomeMessage];
+let messages = [welcomeMessage];
```

Comment:
Why change this to let?

Reply:
thank you for pointing it, I was just playing with different ideas and forgot to put it back.
now fixed it.